x-pack/legacy/plugins/apm/typings/elasticsearch/index.ts

```diff
@@ -4,10 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { Omit } from 'utility-types';
```

Comment:
@dgieselaar I think maybe you added this. You okay with using the `Omit` helper from Typescript instead?s

Reply:
yeah, I think I just copy+pasted the implementation from `utility-types` anyway 😁

Reply:
Yeah, assuming it's the same implementation as from `utility-types`. Not sure, but I guess you will get type errors if it is.